README.md

```diff
@@ -28,8 +28,8 @@ end
 Then start an OTP application:
 
 ```elixir
-defp application do
-  [applications: [:xlsxir]]
+def application do
+  [extra_applications: [:xlsxir]]
 end
```

Comment:
I'm not sure if this is even needed?

Applications is inferred as of Elixir 1.4. See the "Application inference" section in:
https://elixir-lang.org/blog/2017/01/05/elixir-v1-4-0-released/

Also another blog post on the topic linked via StackOverflow:
https://www.amberbit.com/blog/2017/9/22/elixir-applications-vs-extra_applications-guide/